electron/loading/index.html

```diff
@@ -29,12 +29,12 @@
     const { ipcRenderer } = require("electron");
     ipcRenderer.on("response", (event, arg) => {
       if (typeof arg === "string") {
-        if (arg.includes(/Installing/)) {
+        if (arg.includes("Installing")) {
           document.getElementById("text").innerHTML =
             `Installing app dependencies...
             <br />
             This might take a while`;
-        } else if (arg.includes(/Development mode/)) {
+        } else if (arg.includes("Development mode")) {
```

Comment:
Should be "Development Mode"?
<img width="385" alt="image" src="https://github.com/valory-xyz/olas-operate-app/assets/22725775/98d3b9df-964b-4f4e-a742-0e038d19bc5a">

Reply:
or just Development 🤔